outplacement_final_report_ipf_client/models/client_config.py

```diff
@@ -307,7 +307,7 @@ def post_request(self, outplacement):
             for step_id in goal_id.step_ids:
                 step = {
                     "typ": step_id.step_type,
-                    "niva": str(step_id.education_level_id.name) if step_id.education_level_id else "",
+                    "niva": step_id.education_level_id.name_get()[0][1] if step_id.education_level_id else "",
```

Comment:
This function is really long, in the future we should refactor it in to smaller pieces